packages/mux-video/src/CustomVideoElement.js

```diff
@@ -137,13 +162,19 @@ class CustomVideoElement extends HTMLElement {
     return attrs;
   }
 
-  // We need to handle sub-class custom attributes differently from
-  // attrs meant to be passed to the internal native el.
   attributeChangedCallback(attrName, oldValue, newValue) {
-    this.forwardAttribute(attrName, oldValue, newValue);
+    // Initialize the attributes right after construction when they become available.
+    if (!this.#hasAttrCallback && !this.isConnected) {
+      this.#hasAttrCallback = true;
+      this.#init();
+    }
```

Comment:
this logic is here for a bug issued here https://github.com/muxinc/custom-video-element/issues/11